pkg/cluster/util.go

```diff
@@ -337,15 +350,11 @@ func (c *Cluster) credentialSecretNameForCluster(username string, clusterName st
 
 	return c.OpConfig.SecretNameTemplate.Format(
 		"username", strings.Replace(username, "_", "-", -1),
-		"clustername", clusterName,
```

Comment:
Can you add this line again? Then it works with old and new config map. We can then clean this up at a later stage.